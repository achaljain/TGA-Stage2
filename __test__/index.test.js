import React from 'react';
import { render, cleanup } from '@testing-library/react';
import HackerNewsHome from '../pages';

afterEach(cleanup);

it('matches snapshot', () => {
  const { asFragment } = render(<HackerNewsHome />);
  expect(asFragment()).toMatchSnapshot();
});
