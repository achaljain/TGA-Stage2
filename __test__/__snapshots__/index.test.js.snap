// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Hacker News
    </h1>
    <p>
      WIP...
    </p>
  </div>
</DocumentFragment>
`;
